config.yaml

```diff
@@ -1,3 +1,52 @@
 # Copyright 2023 Canonical Ltd.
 # See LICENSE file for licensing details.
-options: {}
+options:
+  webserver_workers:
+    type: int
+    description: The number of webserver worker processes for handling requests.
+  webserver_threads:
+    type: int
+    description: >
+      The number of worker threads for handling requests. Run each webserver worker with the specified number of threads.
+  webserver_keepalive:
+    type: int
+    description: The number of seconds for webserver to wait for requests on a Keep-Alive connection.
```

Comment:
s/The number of seconds for webserver to/Time in seconds to?

Reply:
I'm thinking here about the resulting docs, this seems like simple grammar to me

Reply:
Updated, thanks!